Food_Tracker_App/app/src/main/java/com/example/foodtracker/ui/mealPlan/AddMealPlanDialog.java

```diff
@@ -0,0 +1,48 @@
+package com.example.foodtracker.ui.mealPlan;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.fragment.app.DialogFragment;
+
+import com.example.foodtracker.R;
+
+/**
+ * Dialog that asks user if they would like to create a meal plan or to add a single day to the
+ * current meal plan.
+ */
+public class AddMealPlanDialog extends DialogFragment{
+
+    public static final String CREATE_MEAL_PLAN_TAG = "Create_meal_plan";
```

Comment:
Is this used?

Reply:
Yes, in MealPlanMainScreen in the onAddClick() method